RiotSwiftUI/Modules/Authentication/ForgotPassword/View/AuthenticationForgotPasswordScreen.swift

```diff
@@ -0,0 +1,147 @@
+    
+    @ObservedObject var viewModel: AuthenticationForgotPasswordViewModel.Context
+    
+    // MARK: Views
+    
+    var body: some View {
+        GeometryReader { geometry in
+            VStack {
+                ScrollView {
+                    mainContent
+                        .readableFrame()
+                        .padding(.horizontal, 16)
+                }
+                
+                if viewModel.viewState.hasSentEmail {
+                    waitingFooter
+                        .padding(.bottom, geometry.safeAreaInsets.bottom > 0 ? 20 : 36)
+                        .padding(.horizontal, 16)
+                }
+            }
+        }
+        .background(background.ignoresSafeArea())
+        .toolbar { toolbar }
+        .alert(item: $viewModel.alertInfo) { $0.alert }
+        .accentColor(theme.colors.accent)
+    }
+    
+    @ViewBuilder
+    var mainContent: some View {
+        if viewModel.viewState.hasSentEmail {
+            waitingContent
+        } else {
+            AuthenticationForgotPasswordForm(viewModel: viewModel)
+        }
+    }
+    
+    var waitingContent: some View {
+        VStack(spacing: 36) {
+            waitingHeader
+                .padding(.top, OnboardingMetrics.breakerScreenTopPadding)
+        }
+    }
+    
+    /// The instructions shown whilst waiting for the user to tap the link in the email.
+    var waitingHeader: some View {
+        VStack(spacing: 8) {
+            OnboardingIconImage(image: Asset.Images.authenticationEmailIcon)
+                .padding(.bottom, OnboardingMetrics.breakerScreenIconBottomPadding)
+            
+            OnboardingTintedFullStopText(VectorL10n.authenticationForgotPasswordWaitingTitle)
+                .font(theme.fonts.title2B)
+                .multilineTextAlignment(.center)
+                .foregroundColor(theme.colors.primaryContent)
+                .accessibilityIdentifier("waitingTitleLabel")
+            
+            Text(VectorL10n.authenticationForgotPasswordWaitingMessage(viewModel.emailAddress))
+                .font(theme.fonts.body)
+                .multilineTextAlignment(.center)
+                .foregroundColor(theme.colors.secondaryContent)
+                .accessibilityIdentifier("waitingMessageLabel")
+        }
+    }
+    
+    /// The footer shown whilst waiting for the user to tap the link in the email.
+    var waitingFooter: some View {
+        VStack(spacing: 14) {
+            Button(action: done) {
+                Text(VectorL10n.done)
+            }
+            .buttonStyle(PrimaryActionButtonStyle())
+            .accessibilityIdentifier("doneButton")
+            
+            Button { viewModel.send(viewAction: .resend) } label: {
+                Text(VectorL10n.authenticationForgotPasswordWaitingButton)
+                    .font(theme.fonts.body)
+                    .multilineTextAlignment(.center)
+            }
+            .accessibilityIdentifier("resendButton")
+        }
```

Comment:
Last request: Could this have a spacing of `12` and also vertical padding of `12` on the resend button's label so that the spacing matches [others](https://github.com/vector-im/element-ios/blob/62add618b310f9efd92da04d1ddc1ba64c08bdeb/RiotSwiftUI/Modules/Onboarding/Congratulations/View/OnboardingCongratulationsScreen.swift#L102-L117) in the flow? And then...